adaptive_memory_v4.0.py

```
@@ -0,0 +1,2095 @@
+
+        @field_validator("retry_delay")
+        def check_non_negative_float(cls, v, info):
+            if not isinstance(v, float) or v < 0.0:
+                raise ValueError(f"{info.field_name} must be a non-negative float")
+            return v
+
+        @field_validator("timezone")
+        def check_valid_timezone(cls, v):
+            try:
+                pytz.timezone(v)
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)